### DISCUSSION TITLE
General scheduling

### DISCUSSION THREAD

**Discussion by Sepehrsalimian** [Sepehrsalimian/WebStarter#9] [Q&A] [answered]
How should I schedule my time for effective programming study, practice, and review?
@BCFCODE

**Comment by BCFCODE** [comment]
First of all, tell me about all the activities you did during the day. By the way, you’re sending your message with a lowercase "i" instead of "I". I would suggest that before sending any message, you check its grammar with AI, because we will have professional clients and I want everything to look professional. @Sepehrsalimian

**Reply by Sepehrsalimian** [reply]
I've established a daily routine that includes watching a web design course, practicing exercises, reviewing freeCodeCamp, and engaging with GitHub. I'm eager to hear your thoughts on the effectiveness of this schedule and if there are any areas where I could be more productive or delve deeper into specific topics. @BCFCODE

**Reply by BCFCODE** [reply]
Tell me in this format: 
GitHub tasks: 2 hours
YouTube task: ? hours
Total: ? hours"
@Sepehrsalimian

**Reply by Sepehrsalimian** [reply]
Every day has different hours, and I prefer to have time organized.
Please give me a sample plan for testing and continuing. @BCFCODE

**Reply by BCFCODE** [reply]
Tell me in this format:
GitHub tasks: 2 hours
YouTube task: ? hours
Total: ? hours"
@Sepehrsalimian
Tell me about one of your days in this format, and I’ll hook you up with a weekly schedule!

**Reply by Sepehrsalimian** [reply]
GitHub tasks: 3 hours
YouTube task: 3 hours
Total: 6 hours
without reviews...
@BCFCODE

**Reply by BCFCODE** [reply]
This Week's Game Plan:

GitHub Tasks: 3 hours
YouTube Task: 5 hours
(Be sure to commit and push your changes so I can follow your progress and help build your GitHub history on this project!)
Total Time: 8 hours
@Sepehrsalimian 
Don’t forget to mark this as answer if this is your plan for the week! 😉 If this is your answer mark it @Sepehrsalimian

**Comment by BCFCODE** [comment]
#Task @Sepehrsalimian  Remember to label everything for perfect organization: 
![image](https://github.com/user-attachments/assets/a8d8a713-44e7-4b68-8687-7a15caf68ab4)

**Reply by Sepehrsalimian** [reply]
I added a new label! Check it.
@BCFCODE

**Reply by BCFCODE** [reply]
Cool! @Sepehrsalimian

**Comment by BCFCODE** [comment]
Why you close this?

**Comment by BCFCODE** [comment]
Did you find the answer you were looking for in this topic? @Sepehrsalimian

**Comment by BCFCODE** [comment, marked as answer]
I haven’t heard from you in a while—what’s the latest?  Give me a quick update and let me know what’s been going on! @Sepehrsalimian

**Reply by BCFCODE** [reply]
Don't forget to mark answer! @Sepehrsalimian

**Reply by Sepehrsalimian** [reply]
part 14 CSS
branching training MOSH
@BCFCODE
